README.md

```diff
@@ -65,6 +65,9 @@ on [Filecoin Slack](https://filecoinproject.slack.com/).
 
 6. Check logs with `docker logs -f saturn-node`
 7. Check there are no errors, registration will happen automatically and node will restart once it receives its TLS certificate
+
+If the speedtest value reported by speedtest seems low, you may want to configure SPEEDTEST_EXTRA to point to a local speedtest server (e.g. `SPEEDTEST_EXTRA="-s 6468"`). In most intsances speedtest does a good job of picking "close" servers but for small networks the geo-ip info used may be incorrect.
```

Comment:
would this use a speedtest server from the local network, which would always give unrealistic results? If anything I think the example should show how to point to a specific publicly hosted speedtest server.

Reply:
So `-s 6468` (for example) picks a public speedtest server but a specific one (namely  Tekify Fiber & Wireless        Fremont, CA   )

Reply:
Ah cool, I thought `-s` would pick a local port or something. This context would be great to add to the comment.

Reply:
Sounds good, in the README do you think or run.sh or somewhere else?

Reply:
To me README seems like a good place, since the other places don't really feature documentation (yet)